torch/_functorch/aot_autograd.py

```diff
@@ -1285,7 +1285,9 @@ def inner_fn_with_anomaly(*args):
             warnings.filterwarnings(
                 "ignore", "Anomaly Detection has been enabled."
             )
-            with torch.autograd.detect_anomaly(check_nan=False):
+            with torch.autograd.set_multithreading_enabled(
+                False
+            ), torch.autograd.detect_anomaly(check_nan=False):
```

Comment:
huh, I'm a little surprised we needed this since you already added this context manager to run in the outer-most bit of aot autograd (so I'd expect to already be turned on?) https://github.com/pytorch/pytorch/pull/100982/files#diff-df954bbf954d2dcb81f687876053267ffa4ddb36ed86b7d2bd76319ff2b94416L3136

Reply:
im trying to find the source of a strange bug on CI that can't locally repro: `FakeTensor doesn't have an attribute fake_mode`. Was throwing things at the wall to see if anything fixed it. Still haven't found it yet.